### DISCUSSION TITLE
Can't add premium add-ons repo to artifactory

### DISCUSSION THREAD

**Discussion by jrbecart** [Laravel-Backpack/community-forum#626] [Q&A (Help)]
It appears that adding the premium Backpack add-ons repository to **Artifactory** might not be possible. I don't possess extensive logs or error information since I need to involve our IT teams. However, are you aware of any instances where this has been achieved by your team or one of your clients?

**Comment by pxpm** [comment]
Sorry @jrbecart never used or know about Artifactory (https://jfrog.com/ ?) until today. 

Hope someone can help you with that 🙏

**Reply by jrbecart** [reply]
Yes the jfrog product: [https://jfrog.com/artifactory/](https://jfrog.com/artifactory/)
It's used like a cache layer for security purposes (among other things) by our IT teams.
